<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>438</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_chess_board"/>
   </widget>
   <widget class="QPushButton" name="btn_start_simulation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>START Arduino Simulation</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_pgn_trace">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PGN Trace</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MIDI Out</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_midi_out">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_load_game"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QMenu" name="menu_com_port">
     <property name="font">
      <font>
       <family>Consolas</family>
      </font>
     </property>
     <property name="title">
      <string>COM Port</string>
     </property>
     <widget class="QMenu" name="menu_com_name">
      <property name="title">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QMenu" name="menu_com_baud">
      <property name="title">
       <string>Baud</string>
      </property>
     </widget>
     <addaction name="menu_com_name"/>
     <addaction name="menu_com_baud"/>
    </widget>
    <widget class="QMenu" name="menu_midi">
     <property name="title">
      <string>MIDI</string>
     </property>
    </widget>
    <addaction name="menu_com_port"/>
    <addaction name="menu_midi"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMIDI">
   <property name="text">
    <string>MIDI</string>
   </property>
  </action>
  <action name="action_load_game">
   <property name="text">
    <string>Load Game</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
  </action>
  <action name="actionBaud">
   <property name="text">
    <string>Baud</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionr">
   <property name="text">
    <string>r</string>
   </property>
  </action>
  <action name="actionf">
   <property name="text">
    <string>f</string>
   </property>
  </action>
  <action name="actionttt">
   <property name="text">
    <string>ttt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
